script/Ciel/Day 8/08_01_CIEL08_3B.txt

```diff
@@ -237,17 +238,18 @@ The butter chicken was finger-licking good.
 [sha:5ba48769cc2c8e32b2e66e9dfddc959ec5a7908f]{
 -- Page 267, Offset 18934.
 -- カレーソースはバターが効いているのか辛いのに甘いという絶妙なバランスで、ナンも熱く柔らかく、パン生地を噛みしめるだけで旨みが口内に溢れてくる。
-The curry sauce was an exquisite balance of spiciness and sweetness, thanks perhaps to the butter. The naan was pipping hot and soft so that just by taking a bite of the bread, I could feel the savoriness spreading all over my taste buds.
+// 旨み to "savoriness" here, dunno if umami is common knowledge enough to keep as is
```

Comment:
umami is a common expression in english and is distinct from savoriness. just transliterate it